.github/workflows/lint.yml

```
@@ -0,0 +1,31 @@
+  pull_request:
+    branches: [main]
+  push:
+    branches: [main]
+
+jobs:
+  lint:
+    name: ESLint Check
+    runs-on: ubuntu-latest
+    # Skip if commit is from GitHub Actions (semver bot) or if it's a merge commit from a PR
+    if: |
+      github.actor != 'github-actions[bot]' && 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)